app/components/common/drawer/index.tsx

```diff
@@ -9,53 +9,51 @@ interface DrawerProps {
   toggleDrawer: () => void;
 }
 
-const Drawer: React.FC<DrawerProps> = ({ event, isDrawerVisible, toggleDrawer }) => {
-  return (
-    <div className={`w-full h-full fixed inset-0 ${isDrawerVisible ? '' : 'invisible'}`}>
-      <div
-        onClick={toggleDrawer}
-        data-testid="drawer-background"
-        className={`w-full h-full duration-500 ease-out transition-all inset-0 absolute bg-gray-900 ${
-          isDrawerVisible ? 'opacity-50' : 'opacity-0'
-        }`}
-      ></div>
-      <div
-        onClick={toggleDrawer}
-        className={`pt-20 pb-12 px-12 min-w-[300px] sm:w-full md:w-1/2 lg:w-1/4 bg-white h-full absolute right-0 duration-300 ease-out transition-all flex flex-col justify-between  overflow-scroll ${
-          isDrawerVisible ? '' : 'translate-x-full'
-        }`}
-      >
+const Drawer: React.FC<DrawerProps> = ({ event, isDrawerVisible, toggleDrawer }) => (
+  <div className={`w-full h-full fixed inset-0 ${isDrawerVisible ? '' : 'invisible'}`}>
+    <div
+      data-testid="drawer-background"
+      className={`w-full h-full duration-500 ease-out transition-all inset-0 absolute bg-gray-900 ${
+        isDrawerVisible ? 'opacity-50' : 'opacity-0'
+      }`}
+      onClick={toggleDrawer}
```

Comment:
I have added the click action here, could you ensure you have taken the latest pull

Reply:
@iRohitGaur

Reply:
Hi @harshith-venkatesh my bad, it didnt take the latest pull. I got the latest pull but found another issue. See the rec:


https://github.com/Real-Dev-Squad/website-calendar/assets/10892977/9516ce69-9408-48be-8fb7-4361987a352f

Reply:
Oh wow, I shouldn't have committed this mistake, thanks for the great review. Let me do the required changes